keyboards/jnao/keymaps/via/keymap.c

```diff
@@ -0,0 +1,155 @@
+#include QMK_KEYBOARD_H
+
+enum layer_names {
+    _QWERTY,
+    _LOWER,
+    _RAISE,
+    _FUNCTION,
+    _ADJUST,
```

Comment:
Just a heads up, VIA doesn't support 5 layers. By default, it only supports 4.  You have to increase it manually to support more, and that is only if there is enough space in the eeprom.